iothub/device/src/Authentication/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -145,5 +145,30 @@ internal static string Sign(string requestString, string key)
             using var algorithm = new HMACSHA256(Convert.FromBase64String(key));
             return Convert.ToBase64String(algorithm.ComputeHash(Encoding.UTF8.GetBytes(requestString)));
         }
+
+        internal static string GetSignature(string encodedURI, string key, long expiry)
+        {
+            return Sign($"{encodedURI}\n{expiry}", key);
+        }
+
+        internal static string GetToken(string encodedURI, string key, long expiry = 0, int defaultTimeToLive = 60)
```

Comment:
Seems there is some duplication with existing methods in this class, so now there are two mechanisms to create the tokens. 

Any reason to do not use BuildSignature()?